src/orange/detail/RectArrayInserter.cc

```diff
@@ -46,14 +46,16 @@ RectArrayId RectArrayInserter::operator()(RectArrayInput const& inp)
 
     for (auto ax : range(Axis::size_))
     {
-        std::vector<double> const& grid = inp.grid[to_int(ax)];
+        std::vector<double> grid = inp.grid[to_int(ax)];
         CELER_VALIDATE(grid.size() >= 2,
                        << "grid for " << to_char(ax) << " axis in '"
                        << inp.label << "' is too small (size " << grid.size()
                        << ")");
         CELER_VALIDATE(std::is_sorted(grid.begin(), grid.end()),
                        << "grid for " << to_char(ax) << " axis in '"
                        << inp.label << "' is not monotonically increasing");
+        grid.front() = -std::numeric_limits<real_type>::infinity();
```

Comment:
Add a comment about what this is doing:
```suggestion
        // Suppress the outer grid boundaries to avoid coincident surfaces with other universes
        grid.front() = -std::numeric_limits<real_type>::infinity();
```
And technically I think we should be "bumping" these rather than making them infinite, but this is just as good if we know a priori the user didn't make an error in construction.